boards/arm/stm32f334_disco/pinmux.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2017 I-SENSE group of ICCS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <device.h>
+#include <init.h>
+#include <drivers/pinmux.h>
+#include <sys/sys_io.h>
+
+#include <pinmux/stm32/pinmux_stm32.h>
+
+/* pin assignments for STM32F3DISCOVERY board */
+static const struct pin_config pinconf[] = {
+#ifdef CONFIG_UART_1
+	{STM32_PIN_PA9, STM32F3_PINMUX_FUNC_PA9_USART1_TX},
+	{STM32_PIN_PA10, STM32F3_PINMUX_FUNC_PA10_USART1_RX},
+#endif	/* CONFIG_UART_1 */
+#ifdef CONFIG_UART_2
+	{STM32_PIN_PA2, STM32F3_PINMUX_FUNC_PA2_USART2_TX},
+	{STM32_PIN_PA3, STM32F3_PINMUX_FUNC_PA3_USART2_RX},
+#endif	/* CONFIG_UART_2 */
+#ifdef CONFIG_I2C_1
+	{STM32_PIN_PB8, STM32F3_PINMUX_FUNC_PB8_I2C1_SCL},
+	{STM32_PIN_PB9, STM32F3_PINMUX_FUNC_PB9_I2C1_SDA},
+#endif /* CONFIG_I2C_1 */
```

Comment:
note that PB6,7,8,9 are connected to user LED by default
Could be conflicting with I2C1